recipes/metabolishmm/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Decompress the HMM database
+tar zxvf metabolisHMM_markers_v1.9.tgz
```

Comment:
You’re unpacking the tarball here, but you need to also copy it over to the `$PREFIX` somewhere so that it will be packaged. Is this some data that the tool needs at runtime? If so, you need to figure out where to put it so it will be found.

Reply:
the package looks for it in the package root directory, where it is unpacked. What would be the environment variable corresponding to that? $PREFIX is the root folder for the environment, correct?